### PR TITLE
Hyundai SantaFe CM and Opel Omega B databases added

### PR DESCRIPTION
**Pull request by MonsterPerformance**
Hyundai SantaFe 2007 2.7V6 4AT: 
Now the signals have understandable names and all ones have been checked on my car.
Useless signals are removed at all.

Opel Omega B 2001 3.2V6 5MT:
New database for GM's Vectra B/Omega B with V6 engines.

### CODE REVIEW

File: opel_omega_2001.dbc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    SG_ ABD_Active        :  4|1@0+ (1.0,0.0) [0.0|1.0] "yes/no"  ECU,TCU
+    SG_ TorqueRequestFast : 15|8@0+ (1.0,0.0) [0.0|255] ""  ECU,TCU
+    SG_ TorqueRequestSlow : 31|8@0+ (1.0,0.0) [0.0|255] ""  ECU,TCU
+
+BO_ 384 SAS_Data: 8 SAS
+    SG_ SteeringAngle :  0|16@1- (0.1,0.0) [-3276.8|3276.7] "yes/no"  ECU,TCU
+    SG_ SteeringSpeed : 16| 8@1+ (1.0,0.0) [0.0|255] ""  ECU,TCU
+
+BO_ 416 ECU_Data1: 8 ECU
+    SG_ RPM            : 15|16@0+ (1.0,0.0) [0.0|65535] "rpm"  ABS,ESP,TCU
```
Comment (rusefillc) [inline opel_omega_2001.dbc:22]: two byte field not aligned with bytes? "15" is start in bits, right?

Comment (MonsterPerformance) [reply opel_omega_2001.dbc:22]: It is aligned. But it uses Motorola byte sequence.
You can check it in your Vector tool.
P.S. Yesterday I had eaten the heap of shit when I was creating this database with such byte sequence. It looks weird.(

Comment (MonsterPerformance) [reply opel_omega_2001.dbc:22]: image
It looks so in Vector CANdb++. :)

Comment (rusefillc) [reply opel_omega_2001.dbc:22]: Wow, today I've learned! Thank you.

Comment (MonsterPerformance) [reply opel_omega_2001.dbc:22]: > Wow, today I've learned! Thank you.

So probably it's time to push MR further? Who does it usually here?)

Comment (rusefillc) [reply opel_omega_2001.dbc:22]: Definitely not me, I am with the same board with you waiting for mine to be pushed :)

I guess that opendbc for non-autopilot cases is understandably not on top of the priority for maintainers?

Comment (MonsterPerformance) [reply opel_omega_2001.dbc:22]: > Definitely not me, I am with the same board with you waiting for mine to be pushed :)
> 
> I guess that opendbc for non-autopilot cases is understandably not on top of the priority for maintainers?

R U that guy who usually fights with Maxi on Turbobazar forum? :)

**Comment by AnotherKote** [comment]
@sshane, @adeebshihadeh, what should be done to merge this PR ?

**Comment by MonsterPerformance** [comment]
@adeebshihadeh @nelsonjchen @deanlee could some please tell how to push my request into repo?

**Pull request closed without merging**